docs/CRSF.md

```diff
@@ -2,86 +2,94 @@
 - Tx Ser Link Mode = mavlink
 - Tx Snd RadioStat = off (yes, off!)
 
-Notes: 
-- There are situations in which it can be useful to enable "Tx Snd RadioStat", but you should do this only if you know what you are doing. You really should not need it for this setup.
-- The choice of Tx Ser Baudrate is uncritical and really determined by the user's need. It is recommended to set it to 57600 or higher, as this will provide enough speed for all operation modes (19 Hz, 31 Hz, 50 Hz).
-
+Note: The choice of Tx Ser Baudrate is not critical and really determined by the user's need. It is recommended to set it to 57600 or higher, as this will provide enough speed for all operation modes (19 Hz, 31 Hz, 50 Hz).
 
 ## ArduPilot Setup
 
-Configuration of a serial port for MAVLink v2:
+A basic setup is described in this section which should get one started, further ArduPilot information is available here: [ArduPilot Systems](docs/ARDUPILOT.md)
 
-- SERIALx_BAUD = 57 
-- SERIALx_OPTIONS = 0
+### MAVLink Serial Port
+
+- SERIALx_BAUD:
+    - 57 for 31 Hz, 50 Hz
+    - 38 for 19 Hz
 - SERIALx_PROTOCOL = 2 (important, do not use MAVLink v1!)
+- SERIALx_OPTIONS = 4096 (ignore commands from GCS to change stream rates)
+
+Note: 'x' refers to the serial port of your flight controller
 
-Configuration of MAVLink stream rates:
+### Stream Rates
 
-- SRx_EXT_STATS = 1 (or 2 if you use 31 Hz or 50 Hz mode)
+- SRx_ADSB = 0
+- SRx_EXT_STAT:
+    - 2 for 3 1Hz, 50 Hz
+    - 1 for 19 Hz
 - SRx_EXTRA1 = 4
 - SRx_EXTRA2 = 4
-- SRx_EXTRA3 = 1 (or 2 if you use 31 Hz or 50 Hz mode)
+- SRx_EXTRA3:
+    - 2 for 31 Hz, 50 Hz
+    - 1 for 19 Hz
+- SRx_PARAMS = 50
 - SRx_POSITION = 2
-- SRx_RAW_SENS = 0 (for most of you this one is unimportant, keep it at 0 unless you really need it)
-
-Configuration for CRSF receiver:
+- SRx_RAW_CTRL = 0
+- SRx_RAW_SENS = 0 
+- SRx_RC_CHAN = 0
 
-Setting up ArduPilot for a CRSF receiver can be a bit tricky, as it depends on the flight controller board, and might need BRD_ALT_CONFIG to be set to a specific value. It is best to consult the ArduPilot wiki, or ask in the ArduPilot discussion channel.
+Note: 'x' refers to the MAVLink stream number of the serial port 
 
-For my Matek H743 board the configuration is:
+### CRSF Receiver
 
-- BRD_ALT_CONFIG = 1
 - RC_PROTOCOLS = 536 or 512
-- SERIAL7_BAUD = irrelevant (baud rate is determined by ArduPilot)
-- SERIAL7_OPTIONS = 0
-- SERIAL7_PROTOCOL = 23
-
-Notes:
-- There are more RC options available in the 'RC_OPTIONS' parameter. ([ArduPilot Docs for RC_OPTIONS](https://ardupilot.org/plane/docs/parameters.html#rc-options-rc-options)) 
-- RSSI_TYPE should be set to either 3 or 5. ([ArduPilot Docs for RSSI_TYPE](https://ardupilot.org/plane/docs/parameters.html#rssi-type-rssi-type)) 
+- RSSI_TYPE = 3 or 5
+- SERIALx_BAUD = irrelevant (baud rate is determined by ArduPilot)
+- SERIALx_OPTIONS = 0
+- SERIALx_PROTOCOL = 23
 
+Note: 'x' refers to the serial port of your flight controller
```

Comment:
maybe: 'x' refers to the serial port of your flight controller which is used for CRSF 

I wonder if we want to add a note like "Setting up ArduPilot for CRSF receivers can be tricky. If these settings do niot work pl see xxxxx". I'm happy without though too.

Reply:
Added another note linking to the ArduPilot page and expanded the section to include that CRSF likes a serial port with DMA.